src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
   // threads since last reset
   double time_system = 3;
+
+  // Number of requests received
+  uint64 reqs_received = 4;
```

Comment:
For streaming calls, each of the mesages sent/received is referred to as "FooRequest" or "FooResponse", so I think the naming of the fields is fine.
(maybe reqs_received -> requests_received).
Let's just add comment  that for streaming RPCs, we do mean number of messages, not RPCs.